pymacaroons/utils.py

```diff
@@ -92,3 +93,39 @@ def equals(val1, val2):
     for x, y in zip(val1, val2):
         result |= ord(x) ^ ord(y)
     return result == 0
+
+
+def add_base64_padding(b):
+    '''Add padding to base64 encoded bytes.
+
+    Padding can be removed when sending the messages.
+
+    @param b bytes to be padded.
+    @return a padded bytes.
+    '''
+    return b + b'=' * (-len(b) % 4)
+
+
+def raw_b64decode(s):
+    if '_' or '-' in s:
```

Comment:
go macaroon uses urf_safe where your current version uses standard, like that it keeps compatibility even with previous version of pymacaroons.